src/throttler/throttler.ts

```diff
@@ -140,17 +168,47 @@ export abstract class Throttler<T, R> {
     this.handle(this.nextWaitingTaskIndex());
   }
 
-  async handle(cursorIndex: number): Promise<void> {
```

Comment:
You switched the order that `handle()` and `process()` appear in this file, which is making the diffs with those methods, `stats()`, and `taskName()` hard to review. Can you put them back in their original order?

Reply:
Done